.github/pull_request_template.md

```diff
@@ -0,0 +1,50 @@
+## Description of changes
+
+This PR…
+
+## Checklist
+
+Before merging to `main` and releasing to prod:
+
+- [ ] Tests
+- [ ] Manual tested in React apps
+- [ ] Deployment notes
+- [ ] Approved
+
+## Deployment notes
+
+```md
+# Component changes
+
+## Time Series
+
+- Added X feature to Time Series bar chart
+
+## Leaderboard
+
+- Fixed Y bug to Leaderboard table
+
+## Counter
+
+- Added Z feature to Counter
+
+# Packages changes
+
+## GraphQL
+
+- Added X query to graphql package
+
+# App changes
+
+## React 18 example
+
+- Added X test to React 18 example
+
+## Storybook
+
+- Added Y story for Time Series component
+
+# Rollback strategy
+
+Re-deploy the last successful release.
```

Comment:
There's no real going back besides releasing a new version and running `npm deprecate` on the affected version.

```suggestion
```